shapely/tests/__init__.py

```diff
@@ -0,0 +1,24 @@
+import os
```

Comment:
I'm not clear on the role of this file; it seems like it should all be consolidated into `conftest.py`.  However, I don't have a Windows machine to test on, so I'm not sure of the effect of moving the DLL import to conftest instead of here.

Reply:
I'm thinking to leave this to another PR as it seems out of scope ?

Reply:
I don't think there is value in moving something we should just delete. I would prefer to clean it up at once, and I don't think there is anything in this file that is required.

Reply:
> I don't think there is value in moving something we should just delete. I would prefer to clean it up at once, and I don't think there is anything in this file that is required.

OK. 
What do you advise to do with the following code, delete or move?:
```
# Show some diagnostic information; handy for CI
print("Python version: " + sys.version.replace("\n", " "))
print("GEOS version: " + geos_version_string)
print("Numpy version: " + numpy.version.version)
```

Reply:
resolved as per https://github.com/shapely/shapely/pull/1681#discussion_r1104118419